src/Node/Render.php

```diff
@@ -9,13 +13,31 @@
 use Twig_Node_Expression_Constant;
 use Twig_Node_Include;
 
+/**
+ * Class to create and compile a new 'Render' node Twig template.
+ *
+ * Components are rendered within their own context. The default variables are
+ * set from the component config file if not given explicitly via the tag.
+ * Variants must be referenced with a double hyphen and be configured in the main
+ * component config. – Having a separate variant file is currently not supported.
```

Comment:
This is a good start, but I'd suggest to rephrase the summary line and description with the following in mind:

1. For class-level docs, don't assume that the reader knows anything about internals and how things work and how they relate to each other; all of that should be explained.

1. In general, it's hard to follow the current docs, because a "context" in Fractal means "default variables", whereas Drupal calls that "variables" (template variables).  A "context" in Drupal usually refers to the surrounding scope and environment of the running script; i.e., the execution context.  I'd recommend to always refer to "Fractal context properties" to make that distinction more clear, or simplify "default variables from component definition".

1. I'm purposively trying to avoid the term "configuration" as well, because that is universally associated with the actual site configuration in Drupal.  It is not recommended to refer to configuration unless you actually mean configuration.  Fractal's config files are much rather _component declaration files_ or _component definition files_.

1. I'm relatively sure that there is some terminology from Twig internals that additionally confuses matters even more — such as "node", which probably means "tag" here?

1. How does a Render node differ from an Include node?  It's worth to briefly state the history, why this separate render tag was created.  From a bird-level perspective, that might actually be the best approach for class-level docs.